app/models/subject.rb

```diff
@@ -0,0 +1,25 @@
+class Subject < ApplicationRecord
+  extend Enumerize
+
+  SUBJECT_NAMES = [
+    "Math",
+    "History",
+    "English Language",
+    "Russian Language",
+    "Physics",
+    "Chemistry",
+    "Biology",
+    "Informatics",
+    "Geography",
+    "Litrature",
+    "Social Science",
+    "Physical Education"
+  ].freeze
+
+  has_many :company_subjects, dependent: :destroy
+  has_many :companies, through: :company_subjects
+
+  validates :name, presence: true, uniqueness: true
+
+  enumerize :name, in: SUBJECT_NAMES
```

Comment:
Why are subject names limited to this array? I think there can be many different ones at the university at least